### DISCUSSION TITLE
GitHub Project Settings Custom fields - how to allocate a value

### DISCUSSION THREAD

**Discussion by matjung** [community/community#36474] [Projects and Issues]
https://learn.microsoft.com/en-us/training/modules/plan-agile-github-projects-azure-boards/5-configure
At the moment I am following a tutorial for creating a project with custom fields.
While the tutorial is great, I struggle to find out, where to allocate a value to the newly created custom field.
Creating the fields is working fine.
I am having the issue in both Edge and another browser.
I did not spot any errors in the JavaScript console.
For the moment I assume, there is a knowledge gap on my side.

**Comment by ossanna16** [comment]
Hey there @matjung and thank you for asking a good question! I'm glad you found our community 🙂

**Reply by matjung** [reply]
Do you have the same problem or can you allocate values to custom fields?
